graph/GRAPH.h

```diff
@@ -0,0 +1,120 @@
+#include<iostream>
+#include<vector>
+#include<queue>
+#include<list>
+#include<tuple>
+#include<set>
+
+
+#ifndef GRAPH_h
+#define GRAPH_h
+
+class graph
```

Comment:
`namespace graph` have already used the name `graph`, you will have to rename it or there's going to be a collision